.codeclimate.yml

```diff
@@ -0,0 +1,15 @@
+version: "2"
+plugins:
+  radon:
+    enabled: true
+    config:
+      threshold: "C"
+  pep8:
+    enabled: true
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - python:
+exclude_patterns:
+- tests/**/*
```

Comment:
Well, we should have tests of as good a quality as the rest of our code. Or is it already a lost cause?

Reply:
Fair enough, will remove this.